R/ppc-loo.R

```diff
@@ -173,8 +173,10 @@ ppc_loo_pit_overlay <- function(y,
                                 trim = FALSE,
                                 adjust = 1,
                                 kernel = "gaussian",
+                                bounds = NULL,
```

Comment:
I don't think we need `bounds` for `ppc_loo_pit_overlay()` since it's always bounded between 0 and 1 and I don't think it's particularly useful to restrict it even further.